Emergence/Models/Electric Objects/Image.swift

```diff
@@ -2,14 +2,22 @@ import Gloss
 import Foundation
 import CoreGraphics
 
-struct Image: Imageable, ImageURLThumbnailable {
+struct Image: Imageable, ImageURLThumbnailable, GeminiThumbnailable  {
     let id: String
     let imageFormatString: String
     let imageVersions: [String]
     let imageSize: CGSize
     let aspectRatio: CGFloat?
+    let geminiToken: String?
 
     let isDefault: Bool
+
+    func bestThumbnailWithHeight(height:CGFloat) -> NSURL? {
+        if let url = geminiThumbnailURLWithHeight(Int(height)) {
+            return url
+        }
+        return bestAvailableThumbnailURL()
```

Comment:
its a part of the `Thumbnailable` protocol : https://github.com/artsy/Emergence/blob/master/Emergence/Models/Artsy%20Model%20Protocols/Thumbnailable.swift